pkg/cmd/server/admin/overwrite_bootstrappolicy.go

```diff
@@ -191,9 +197,12 @@ func OverwriteBootstrapPolicy(optsGetter restoptions.Getter, policyFile, createB
 			case *authorizationapi.RoleBinding:
 				ctx := kapi.WithNamespace(kapi.NewContext(), t.Namespace)
 				if change {
-					roleBindingStorage.Delete(ctx, t.Name, nil)
-					if _, err := roleBindingStorage.CreateRoleBindingWithEscalation(ctx, t); err != nil {
-						return err
+					_, _, err := roleBindingStorage.UpdateRoleBindingWithEscalation(ctx, t)
```

Comment:
This isn't functionally equivalent.  An update can't change immutable fields (`roleRef`), but deleting and recreating can.